packages/http/libraries/core/src/http/models/MiddlewareHandler.ts

```diff
@@ -1,10 +0,0 @@
-export type MiddlewareHandler<
-  TRequest,
-  TResponse,
-  TNextFunction,
-  TResult = void,
-> = (
-  req: TRequest,
-  res: TResponse,
-  next: TNextFunction,
-) => Promise<TResult> | TResult;
```

Comment:
I would keep the type. They look the same but they are not the same.

Reply:
Sure, don't merge, I will readd it